env.default

```diff
@@ -58,6 +58,18 @@ BASKET_SQS_QUEUE_URL=
 # See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
 BASKET_API_ROOT_URL=https://basket-dev.allizom.org/
 
+# CSP config
+# Default values for CSP items can be found in "donate/settings/defaults.py"
+CSP_DEFAULT_SRC=
+CSP_SCRIPT_SRC=
+CSP_IMG_SRC=
+CSP_FONT_SRC=
+CSP_STYLE_SRC=
+CSP_FRAME_SRC=
+CSP_CONNECT_SRC=
+CSP_BASE_URI=
+CSP_WORKER_SRC=
```

Comment:
we don't need these empty strings now. 
```suggestion
```

Reply:
Hi Pomax! Thanks for the heads up, I have removed them, and resubmitted for review 👍